### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by ewabuczkowska**
Hi @KoenMeecham, could you please review my code for this prac? Thank you!

### CODE REVIEW

File: prac_09/silver_service_taxi.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""
+CP1404  - Practical 09 - Silver Service Taxi class
+"""
+
+from prac_09.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    flagfall = 4.50
```
Comment (KoenMeecham) [inline prac_09/silver_service_taxi.py:9]: Add a docstring under the class and init

File: prac_09/taxi_simulator.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    total_bill = 0
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2),
+             SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    print("Let's drive!")
+    print(MENU)
+    menu_choice = input(">>> ").lower()
+    while menu_choice != "q":
+        if menu_choice == "c":
+            print("Taxis available: ")
```
Comment (KoenMeecham) [inline prac_09/taxi_simulator.py:22]: I would split this into a separate function

File: prac_09/taxi_simulator.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            display_taxis(taxis)
+            # no error-checking
+            taxi_choice = int(input("Choose taxi: "))
+            try:
+                current_taxi = taxis[taxi_choice]
+            except IndexError:
+                print("Invalid taxi choice")
+        elif menu_choice == "d":
+            if current_taxi:
```
Comment (KoenMeecham) [inline prac_09/taxi_simulator.py:31]: I would also refactor this into a function

File: prac_09/taxi_test.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+"""
+CP1404  - Practical 09 - 1. Test Taxi
+"""
+from prac_09.taxi import Taxi
+
```
Comment (KoenMeecham) [inline prac_09/taxi_test.py:5]: You could add a docstring explaining the tests

File: prac_09/unreliable_car.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+"""
+from prac_09.car import Car
+import random
+
+
+class UnreliableCar(Car):
+    """Version of car that includes reliability."""
+
+    def __init__(self, name, fuel, reliability):
+        super().__init__(name, fuel)
```
Comment (KoenMeecham) [inline prac_09/unreliable_car.py:12]: Add some docstrings under the init and drive methods

File: prac_09/unreliable_car_test.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+"""
+CP1404  - Practical 09 - 2. Unreliable car
+"""
+from prac_09.unreliable_car import UnreliableCar
+
```
Comment (KoenMeecham) [inline prac_09/unreliable_car_test.py:5]: Maybe add a docstring explaining the tests

**Comment by ewabuczkowska** [comment]
Thank you for your advice @KoenMeecham :)

**Pull request merged**